packages/mobile/src/escrow/saga.ts

```diff
@@ -324,6 +324,7 @@ function* withdrawFromEscrow(komenciActive: boolean = false) {
     }
 
     yield put(fetchStableBalances())
```

Comment:
Do we still need thise?

Reply:
We do for places that use the old balance, like the Celo tab or the add/withdraw screen :/